stylelint-config.json

```diff
@@ -1,5 +1,59 @@
 {
   "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
```

Comment:
Man, now I won't be able use to [`chucknorris`](http://stackoverflow.com/questions/8318911/why-does-html-think-chucknorris-is-a-color) as a color any more.